.travis.yml

```diff
@@ -46,6 +46,8 @@ jobs:
       dep ensure -v -vendor-only
       cd ../..
       goreleaser --snapshot --skip-sign
+      cd cmd/exo
+      go run ./completion/main.go
```

Comment:
the goal would be to include those files in the release, or do I miss something?

Reply:
yes